fixtures/examples/api-elements/instagram.json

```diff
@@ -414,7 +414,67 @@
                     {
                       "element": "httpTransaction",
                       "meta": {},
-                      "attributes": {},
```

Comment:
I see changes to the api-elements version, but no changes to the underlying Swagger document. Should these be here?